types/google-apps-script/google-apps-script.types.d.ts

```diff
@@ -8,6 +8,6 @@ declare module GoogleAppsScript {
   type Byte = number;
   type Integer = number;
   type Char = string;
-  type String = string;
+  type String = string;// Should be unnecessary now that I replaced all String with string
```

Comment:
I think you could delete this whole line, or delete the comment as users may not understand who is "I" mentioned here 😃

Reply:
Please delete this 😨